### PR TITLE
OCPBUGS#8290: Amended outdated reference to Configuring Fluentd

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.14+

Issue:
[OCPBUGS-8290](https://issues.redhat.com/browse/OCPBUGS-8290)

Link to docs preview:
https://68616--docspreview.netlify.app/openshift-rosa/latest/logging/cluster-logging-deploying

QE review:
No QE review necessary as it is a simple text change.

### CODE REVIEW

**Comment by AedinC** [comment]
\label peer-review-needed

File: modules/cluster-logging-deploy-console.adoc [hunk 1]
```diff
@@ -145,7 +145,7 @@ However, an unmanaged deployment does not receive updates until OpenShift Loggin
 <7> Specify the CPU and memory requests for Elasticsearch as needed. If you leave these values blank, the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `16Gi` for the memory request and `1` for the CPU request.
 <8> Specify the CPU and memory requests for the Elasticsearch proxy as needed. If you leave these values blank, the OpenShift Elasticsearch Operator sets default values that should be sufficient for most deployments. The default values are `256Mi` for the memory request and `100m` for the CPU request.
 <9> Settings for configuring Kibana. Using the CR, you can scale Kibana for redundancy and configure the CPU and memory for your Kibana nodes. For more information, see *Configuring the log visualizer*.
-<10> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits. For more information, see *Configuring Fluentd*.
+<10> Settings for configuring Fluentd. Using the CR, you can configure Fluentd CPU and memory limits. For more information, see *Configuring the logging collector*.
```
Comment (dfitzmau) [inline modules/cluster-logging-deploy-console.adoc:148]: `Settings for configuring Fluentd:` would provide a better heading lead in to this call out. At the moment this reads as an incomplete sentence. 

I know you cannot add an xref to a module, but maybe creating an additional resources on the assembly and just after the module would provide a space to link to the `Configuring the logging collector` section? At the moment the module is buried in the TOC. 

Maybe stating ` see the section named "Configuring the logging collector"` would be clearer that a customer needs to find a module. I initially thought a callout heading existed for `Configuring the logging collector`.

Comment (AedinC) [reply modules/cluster-logging-deploy-console.adoc:148]: Thanks @dfitzmau. That makes a lot of sense, and solves the non-linking issue, which must be annoying for  the user.

**Pull request merged**